### DISCUSSION TITLE
ShiftEnter plugin to enable soft return/enter

### DISCUSSION THREAD

**Discussion by MisterMike** [craftcms/ckeditor#102] [Ideas] [answered]
Consider a simple Richttext (CKEditor) field for an address block that has the address on multiple lines.

Coming from Redactor, I am used to press shift-Enter to add soft returns with a BR html tag. In the current CKEditor version, this does not work and any line breaks are transformed into new paragraphs P.

Would love to see the [ShiftEnter Plugin](https://ckeditor.com/docs/ckeditor5/latest/api/module_enter_shiftenter-ShiftEnter.html) in the core version, if that is something you could consider.

thank you :-)

**Comment by brandonkelly** [comment, marked as answer]
ShiftEnter is included by the Essentials plugin, which is used by every CKEditor 5 build. Just verified it’s working on my end, even with a bare-bones toolbar config.
